Bottom: b4394255d33d01ddf36812ba35ef57f790d90f0c
Top:    c99c995abee2e9fcd1817d712da3b08d85d66c75
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-30 05:39:22 +0100

Refresh of convert-board-objects-to-glist

---

diff --git a/src/autoroute.c b/src/autoroute.c
index f98019c..27dba28 100644
--- a/src/autoroute.c
+++ b/src/autoroute.c
@@ -159,10 +159,6 @@ static hidGC ar_gc = 0;
 	 CONFLICT_LEVEL(rb)==LO_CONFLICT ? \
 	 AutoRouteParameters.LastConflictPenalty : 1) * (rb)->pass)
 
-#if !defined(ABS)
-#define ABS(x) (((x)<0)?-(x):(x))
-#endif
-
 #define _NORTH 1
 #define _EAST 2
 #define _SOUTH 4
diff --git a/src/crosshair.c b/src/crosshair.c
index 2abe960..c2551ec 100644
--- a/src/crosshair.c
+++ b/src/crosshair.c
@@ -56,10 +56,6 @@
 
 RCSID ("$Id$");
 
-#if !defined(ABS)
-#define ABS(x) (((x)<0)?-(x):(x))
-#endif
-
 typedef struct
 {
   int x, y;
diff --git a/src/edif.y b/src/edif.y
index da33c9b..cb31d9a 100644
--- a/src/edif.y
+++ b/src/edif.y
@@ -38,7 +38,6 @@ LibraryEntryTypePtr GetLibraryEntryMemory (LibraryMenuTypePtr);
 #define	IDENT_LENGTH		255
 #define	Malloc(s)		malloc(s)
 #define	Free(p)			free(p)
-#define	ABS(v)			((v) < 0 ? -(v) : (v))
 #define	Getc(s)			getc(s)
 #define	Ungetc(c)		ungetc(c,Input)
 
diff --git a/src/mtspace.c b/src/mtspace.c
index 9d593e8..40f5413 100644
--- a/src/mtspace.c
+++ b/src/mtspace.c
@@ -55,7 +55,6 @@
 #include <dmalloc.h>
 #endif
 
-#define ABS(x) ((x) < 0 ? -(x) : (x))
 RCSID ("$Id$");
 
 /* mtspace data structures are built on r-trees. */
diff --git a/src/polygon1.c b/src/polygon1.c
index 0a3a2b5..1dba74c 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -54,9 +54,6 @@
 
 #define EPSILON (1E-8)
 #define IsZero(a, b) (fabs((a) - (b)) < EPSILON)
-#ifndef ABS
-#define ABS(x) ((x) < 0 ? -(x) : (x))
-#endif
 
 /*********************************************************************/
 /*              L o n g   V e c t o r   S t u f f                    */
